src/components/cardboard-controls.js

```diff
@@ -0,0 +1,34 @@
+const CARDBOARD_BUTTON = "Cardboard Button";
```

Comment:
I was confused about what this was for until I got to the part in the code (I assumed it was basically for a UI string) perhaps rename to `CARDBOARD_BUTTON_GAMEPAD_ID`?